grammars/javascript.cson

```diff
@@ -341,6 +341,31 @@
     ]
   }
   {
+    'comment': 'match stuff like: Sound.play = (a,b) => { … }'
+    'name': 'meta.function.arrow.js'
+    'begin': '([a-zA-Z_?$]*)\\.([a-zA-Z_?$][\\w?$]*)\\s*(=)\\s*(\\()(?=(?:[^\\(\\)]*)?\\)\\s*=>)'
+    'beginCaptures':
+      '1':
+        'name': 'support.class.js'
+      '2':
+        'name': 'entity.name.function.js'
+      '3':
+        'name': 'keyword.operator.js'
+      '4':
+        'name': 'punctuation.definition.parameters.begin.js'
+    'end': '(\\))(\\s*=>)'
+    'endCaptures':
+      '1':
+        'name': 'punctuation.definition.parameters.end.js'
+      '2':
+        'name': 'storage.type.arrow.js'
```

Comment:
This should be addressed in a different PR (after this one is merged) to maintain consistency with the other rules.